core/src/main/java/com/hcl/ecomm/core/listeners/package-info.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright 2017 Adobe Systems Incorporated
+ *  Copyright 2015 Adobe Systems Incorporated
```

Comment:
@DeepaliAgrwl : This merge is wrong 
ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/breadcrumb/clientlib/less/breadcrumb.less → core/src/main/java/com/hcl/ecomm/core/listeners/package-info.java

Reply:
@rishabh-behl I have removed this file. Please do let us know if this merge error still persists.